_includes/head.html

```diff
@@ -3,8 +3,13 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge" />
   <meta name="viewport" content="width=device-width, initial-scale=1" />
   {% if page.layout == "eip" %}
-    <title>EIP-{{ page.eip }}: {{ page.title | escape }}</title>
-    <meta property="og:title" content="EIP-{{ page.eip }}: {{ page.title | escape }}" />
+    {% if page.category == "ERC" %}
+      <title>EIP-{{ page.eip }}: {{ page.title | escape }}</title>
+      <meta property="og:title" content="EIP-{{ page.eip }}: {{ page.title | escape }}" />
+    {% elsif page.category != "ERC" %}
+      <title>ERC-{{ page.eip }}: {{ page.title | escape }}</title>
```

Comment:
/me is still opposed to this change >:(

Reply:
Wait, hold up. I might have reversed the logic (and introduced a typo)
![image](https://user-images.githubusercontent.com/45835846/208244309-d0784cdc-fc80-4c83-88e7-952830d031f5.png)
![image](https://user-images.githubusercontent.com/45835846/208244315-1254ccf2-8461-4425-b19d-e3ba7143297d.png)